components/constants/network_constants.h

```diff
@@ -6,6 +6,8 @@
 #ifndef BRAVE_COMPONENTS_CONSTANTS_NETWORK_CONSTANTS_H_
 #define BRAVE_COMPONENTS_CONSTANTS_NETWORK_CONSTANTS_H_
 
+#include "brave/browser/widevine/buildflags.h"
```

Comment:
Thank you for pointing this out. I'm not really familiar with the layers and allowed dependencies between them.

This PR needs the build flag `WIDEVINE_ARM64_DLL_FIX` in the following places:

- `components`
- `browser`
- `base/threading`

Is there a place / layer where we can define the build flag only once? I would much prefer not having to duplicate it.

Reply:
pls move buildflag definition into `components`.
`base` violation is kinda acceptable, because we don't introduce a build dependency (the new header is static and is ready to be used).

Reply:
Fixed in 657436d07e26825257cbfb55c644a805f46f7e33 ("Fix layer violation").